README.md

```diff
@@ -58,6 +58,16 @@ OR (if running `ino_web_server.py`):
 1. Open browser to BlocklyDuino, drag and drop blocks to make an Arduino program.
 2. Select the 'Arduino' tab and press the 'Upload' button. (press the 'Reset' button to upload an empty program)
 
+### Developpers
+To work on BlocklyDuino, you need google blockly, google closure and python
+
+- Clone the repository with submodules : `git clone --recurse-submodules https://github.com/BlocklyDuino/BlocklyDuino`
+- Checkout the v2 branch : `git checkout v2`
```

Comment:
We can do clone in 2 steps as well
1. Clone the repository `git clone https://github.com/BlocklyDuino/BlocklyDuino`
2. Update submodules `git submodule update --init`